### PR TITLE
Reorg bulk

### PR DESCRIPTION
**Pull request by delvedor**
Related: #409.

### CODE REVIEW

File: specification/_global/bulk/types.ts [hunk 1]
```diff
@@ -56,3 +63,27 @@ export class BulkCreateResponseItem extends BulkResponseItemBase {}
...
+export class BulkOperation {
+  _id: Id
+  _index: IndexName
+  retry_on_conflict: integer
+  routing: Routing
+  version: VersionNumber
+  version_type: VersionType
+}
+
+export class BulkOperationContainer {
```
Comment (philkra) [inline specification/_global/bulk/types.ts:76]: don't we need to annotate this with `/** @variants container */` ?

**Review by philkra — APPROVED** [review]
one question, otherwise LGTM

File: specification/_global/bulk/types.ts (hunk 1 above)
Comment (delvedor) [reply specification/_global/bulk/types.ts:76]: Great catch!

**Review by philkra — APPROVED** [review]
LGTM

**Pull request merged**